homeassistant/components/aurora_abb_powerone/config_flow.py

```diff
@@ -56,55 +89,109 @@ def validate_and_connect(
     return ret
 
 
-def scan_comports() -> tuple[list[str] | None, str | None]:
-    """Find and store available com ports for the GUI dropdown."""
-    com_ports = serial.tools.list_ports.comports(include_links=True)
-    com_ports_list = []
-    for port in com_ports:
-        com_ports_list.append(port.device)
-        _LOGGER.debug("COM port option: %s", port.device)
-    if len(com_ports_list) > 0:
-        return com_ports_list, com_ports_list[0]
-    _LOGGER.warning("No com ports found.  Need a valid RS485 device to communicate")
-    return None, None
+def __validate_and_connect_tcp(hass: core.HomeAssistant, data):
```

Comment:
Not sure why double underscore prefix is needed.  I think single underscore is ok here.